frontend/src/constants/SideNavConfig.ts

```diff
@@ -0,0 +1,23 @@
+import type { SideNavItemType } from "@/types/SideNavTypes";
+import UserMultiple from "@carbon/icons-vue/es/user--multiple/16";
+import IbmLpa from "@carbon/icons-vue/es/ibm--lpa/16";
+
+export const sideNavItems: SideNavItemType[] = [
+    {
+        name: "Manage permissions",
+        link: "/manage-permissions",
+        icon: UserMultiple,
+        childLinks: [
+            "/user-details",
+            "/grant",
+            "/grant-app-admin",
+            "/grant-delegated-admin",
+        ],
+    },
+    {
+        name: "My permissions",
+        link: "/my-permissions",
+        icon: IbmLpa,
+        disabled: false,
```

Comment:
imo disabled should have the same meaning and function as an html's input, so in this case it should make things greyed out and non-clickable. I left this in here as a legacy property from the previous JSON design, it might be useful, although i prefer readonly over disabled since it's more a11y friendlly

Reply:
oh i see what you meant, it has a display: none style at the bottom, that's bad, let me change it

Reply:
https://github.com/bcgov/nr-forests-access-management/pull/1633/commits/dbc598930c6fa64fee39119a4dd316f1d211cb29